pkg/detector/detector.go

```diff
@@ -206,6 +191,19 @@ func (d *ResourceDetector) Reconcile(key util.QueueKey) error {
 	}
 	klog.Infof("Reconciling object: %s", clusterWideKey)
 
+	if clusterWideKey.Group == policyv1alpha1.GroupName {
+		switch clusterWideKey.Kind {
+		case "PropagationPolicy":
+			return d.ReconcilePropagationPolicy(key)
+		case "ClusterPropagationPolicy":
+			return d.ReconcileClusterPropagationPolicy(key)
+		}
+	}
+
+	if !d.EventFilter(clusterWideKey) {
+		return nil
+	}
+
```

Comment:
The `if clusterWideKey.Group == policyv1alpha1.GroupName` block better to moved into the `if !d.EventFilter(clusterWideKey)`, like:
```
	if !d.EventFilter(clusterWideKey) {
		if clusterWideKey.Group == policyv1alpha1.GroupName {
			switch clusterWideKey.Kind {
			case "PropagationPolicy":
				return d.ReconcilePropagationPolicy(key)
			case "ClusterPropagationPolicy":
				return d.ReconcileClusterPropagationPolicy(key)
			}
		}
		klog.V(4).Infof("Ignored object(%s) from reconciling.", clusterWideKey.String())
		return nil
	}
	klog.Infof("Reconciling object: %s", clusterWideKey)
```